common/archive-impl/impl2/src/webapp/WEB-INF/components.xml

```diff
@@ -23,6 +23,7 @@
 				<value>AssignmentService</value>
 				<value>AssessmentEntityProducer</value>
 				<value>ContentHostingService</value>
+				<value>conversations</value>
```

Comment:
Why is this not "ConversationsServiceImpl" - is there more than one way to add a service to the list (i.e. by service name)?

Reply:
It's what EntityProducer.getLabel returns.